src/template/mapping.json

```diff
@@ -348,7 +348,9 @@
   "text-size": 60265,
   "three-bars": 60266,
   "thumbsdown": 60267,
+  "thumbsdown-filled": 60267,
```

Comment:
These should go at the very bottom of the list and use unique values. I just add 1 to whatever the last icon was. So in this case:

```
"mic": 60434,
"thumbsdown-filled": 60435,
"thumbsup-filled": 60436
```

Then rerun the build task and you should be good to go

Reply:
Fixed!